examples/website/contour/app.js

```diff
@@ -0,0 +1,130 @@
+import React, {PureComponent, Fragment} from 'react';
+import {render} from 'react-dom';
+import {StaticMap} from 'react-map-gl';
+import {ContourLayer} from '@deck.gl/aggregation-layers';
+import DeckGL from '@deck.gl/react';
+import RangeSlider from './rangeSlider';
+
+// Set your mapbox token here
+const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
+
+// Source data CSV
+const DATA_URL =
+  'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv'; // eslint-disable-line
+
+const INITIAL_VIEW_STATE = {
+  longitude: -1.4157267858730052,
+  latitude: 52.232395363869415,
+  zoom: 6.6,
+  minZoom: 5,
+  maxZoom: 15,
+  pitch: 40.5,
+  bearing: -27.396674584323023
+};
+
+const THRESHOLD = {
+  MIN: 1,
+  MAX: 20
+};
+
+export class App extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      thresholdValue1: 5,
+      thresholdValue2: 10,
+      cellSize: 1000
+    };
+    this.handleCellSizeChange = this.handleCellSizeChange.bind(this);
+    this.onThresholdValueChange = this.onThresholdValueChange.bind(this);
+  }
+
+  onThresholdValueChange(thresholdValue1, thresholdValue2) {
+    this.setState({
+      thresholdValue1,
+      thresholdValue2
+    });
+  }
+
+  handleCellSizeChange(event) {
+    const cellSize = event.target.value;
+    this.setState({cellSize});
+  }
+
+  _renderLayers() {
+    const {data, coverage = 1} = this.props;
+    const {thresholdValue1, thresholdValue2, cellSize} = this.state;
+    return [
+      new ContourLayer({
+        id: 'contourlayer',
+        coverage,
```

Comment:
there no `coverage` prop, can be removed.

Reply:
Done